src/promptflow/promptflow/_sdk/data/executable/main_csharp.py.jinja2

```diff
@@ -0,0 +1,140 @@
+import json
+import os
+from pathlib import Path
+from PIL import Image
+import streamlit as st
+from streamlit_quill import st_quill
+
+from promptflow import load_flow
+
+from utils import dict_iter_render_message, parse_list_from_html, parse_image_content
+
+
+{% set indent_level = 4 %}
+
+def start():
+    def clear_chat() -> None:
+        st.session_state.messages = []
+
+    def render_message(role, message_items):
+        with st.chat_message(role):
+            dict_iter_render_message(message_items)
+
+    def show_conversation() -> None:
+        if "messages" not in st.session_state:
+            st.session_state.messages = []
+            st.session_state.history = []
+        if st.session_state.messages:
+            for role, message_items in st.session_state.messages:
+                render_message(role, message_items)
+
+
+    def get_chat_history_from_session():
+        if "history" in st.session_state:
+            return st.session_state.history
+        return []
+
+
+    def submit(**kwargs) -> None:
+        st.session_state.messages.append(("user", kwargs))
+        session_state_history = dict()
+        session_state_history.update({"inputs": kwargs})
+        with container:
+            render_message("user", kwargs)
+        # Force append chat history to kwargs
+{% if is_chat_flow %}
```

Comment:
can we remove jinja this time?

Reply:
There are several issued I'm not sure.
1. If use python, we need update StreamlitFileGenerator? since it inherit BaseGenerator which ony support render jinja file.
2. Not sure whether python can support jinja logic here，![image](https://github.com/microsoft/promptflow/assets/26239730/f1836b8f-b9e9-4469-8bef-28673bccc73d)

We need generate in inputs box for every flow input. Suppose we have two inputs here: input1, input2. Streamlit need codes like:
```
input1 = = st.file_uploader(label='input1 ')
input2 = st.text_input(label='input2', placeholder=default_value)
```
But how it could be achieved by python even though we use if else here
![image](https://github.com/microsoft/promptflow/assets/26239730/8d301171-254a-4fe1-a7e3-216b1fa6057c)

Reply:
update: issue 2 can resolve, but about issue1, how can we pass those param to python file.
It seems  we can only run this script by ```streamlit run main.py```, so we can't pass params in cmd.

![image](https://github.com/microsoft/promptflow/assets/26239730/0434f3d4-8ddd-491d-9c56-3f1385b3fff7)

Reply:
removed jinja